tools/build_openblas.sh

```diff
@@ -1,40 +1,62 @@
 #!/bin/bash
 # Build script for OpenBLAS on Windows
-# Expects environment variables:
-#  OPENBLAS_ROOT
-#  OPENBLAS_COMMIT
-#  BUILD_BITS
-#  START_DIR
-# Expects "lib.exe" and "gcc" to be on the path
+#
+# Usage: build_openblas.sh [openblas_root [build_bits [if_bits]]]
+#
+# e.g build_openblas.sh c:\\opt 64 32
+#
+# Uses the optional environment variables.  We always prefer command line argument
+# values above to environment variable values:
+#
+#  OPENBLAS_ROOT  (default directory root for binaries, unspecified -> c:\opt).
+#  BUILD_BITS  (default binary architecture, 32 or 64, unspec -> 64).
+#  INTERFACE64  (1 for 64-bit interface, anything else or undefined for 32,
+#                This gives the default value if if_bits not specified above).
```

Comment:
It would be nice to make the connection between the command line `if_bits` and the `INTERFACE64` environment variable more explicit. What happens if I specify `if_bits==32` like in the example above?

Reply:
The comment above was meant to cover that, unless I misunderstood:

```
# Uses the optional environment variables. We always prefer command line argument
```